COLLADA2GLTF/extensions/o3dgc-compression/GLTF-Open3DGC.cpp

```diff
@@ -326,8 +326,8 @@ namespace GLTF
             
             for (size_t j = 0 ; j < attributesCount ; j++) {
                 shared_ptr <GLTFAccessor> meshAttribute = mesh->getMeshAttribute(semantic, j);
-                vertexCount = meshAttribute->getCount();
-                size_t componentsPerElement = meshAttribute->componentsPerElement();
+                vertexCount = (long)meshAttribute->getCount();
```

Comment:
You shouldn't need to cast here because there is no loss of precision from size_t -> long.